admin_guide/pruning_resources.adoc

```diff
@@ -185,6 +185,23 @@ streams. Images are non-namespaced resources. Therefore, limiting pruning to a
 particular namespace makes it impossible to calculate their current usage.
 ====
 
+It's important to know that by default the integrated registry caches blobs metadata
+to reduce the number of requests to the storage and increase the speed of processing
+the request. Pruning doesn't update the integrated registry cache. It means that images
+that will be pushed after pruning and that contain pruned layers will be broken.
+So do this after the pruning is complete. You can run to wipe the cache:
```

Comment:
"Therefore it is necessary to clear the cache after pruning.  This can be accomplished by redeploying the registry:"